src/lib/Tab/LocationView/LocationsTab.php

```diff
@@ -28,6 +34,7 @@
 class LocationsTab extends AbstractEventDispatchingTab implements OrderedTabInterface
 {
     const URI_FRAGMENT = 'ez-tab-location-view-locations';
+    private const PAGINATION_PARAM_NAME = 'locationstab-page';
```

Comment:
Not sure what is the correct convention here but `locations-tab` looks really bad. `locations-tab-page` is better but please check other places and keep it streamlined with the codebase.